.gitlab/pipelines/stages/publish.sh

```diff
@@ -2,10 +2,19 @@
 
 set -o errexit -o nounset
 
+version=$(cat "output/version.txt")
+
+if [ -z "${version}" ]; then
+  >&2 echo "Version not set!"
+  exit 1
+fi
+
 mc config host add "s3" "https://s3.amazonaws.com" "${AWS_ACCESS_KEY}" "${AWS_SECRET_KEY}"
 
-mc cp output/kubecf-*.tgz "s3/scf-v3"
-echo "URL: https://scf-v3.s3.amazonaws.com/$(basename output/kubecf-*.tgz)"
+mc cp "output/kubecf.tgz" "s3/scf-v3/kubecf-${version}.tgz"
+echo "URL: https://scf-v3.s3.amazonaws.com/kubecf-${version}.tgz"
 
-mc cp output/bundle-*.tgz "s3/kubecf"
-echo "URL: https://kubecf.s3.amazonaws.com/$(basename output/bundle-*.tgz)"
+mc cp "output/kubecf.tgz" "s3/kubecf/kubecf-${version}.tgz"
+echo "URL: https://kubecf.s3.amazonaws.com/kubecf-${version}.tgz"
```

Comment:
Huh, we're copying it to `scf-v3` _and_ `kubecf`? Never mind that previous comment then, I'm just plain confused now :D

Reply:
The idea is that we deprecate scf-v3, and start telling people to use kubecf.